monai/transforms/spatial/array.py

```diff
@@ -3262,7 +3262,7 @@ def filter_threshold(self, image_np: NdarrayOrTensor, locations: np.ndarray) ->
 
     def filter_count(self, image_np: NdarrayOrTensor, locations: np.ndarray) -> tuple[NdarrayOrTensor, np.ndarray]:
         """
-        Sort the patches based on the sum of their intensity, and just keep `self.num_patches` of them.
+        Sort the patches based on the sum of their intensity or in random order, and just keep `self.num_patches` of them.
```

Comment:
Could you please help also update the docstring for the `sort_fn` in Line3199?
And do we also need update `RandGridPatch` since it's inherit from the `GridPatch`.